app/Http/Controllers/Api/DepartmentsController.php

```diff
@@ -15,7 +15,7 @@ class DepartmentsController extends Controller
     /**
      * Display a listing of the resource.
      *
-     * @author [A. Gianotto] [<snipe@snipe.net>]
+     * @author [Godfrey Martinez] [<snipe@snipe.net>]
```

Comment:
Your email address too :P I don't need to start getting emails for you to my personal account :P

Reply:
(You do not need to share your email address if you don't want to, but if you don't, you should remove that part and not include an email address, versus using mine.)